site/en/docs/extensions/reference/storage/index.md

```diff
@@ -2,26 +2,66 @@
 api: storage
 ---
 
-## Overview
+## Overview {: #overview }
 
-This API has been optimized to meet the specific storage needs of extensions. It provides the same
-storage capabilities as the [localStorage API][local-storage] with the following key differences:
+The Storage API provides an extension-specific way to persist user data and state. It's similar to the web platform's storage APIs ([IndexedDB][mdn-indexeddb], [localStorage][mdn-localstorage]), but was designed to meet the storage needs of extensions. The following are a few key features: 
 
-- User data can be automatically synced with Chrome sync (using `storage.sync`).
-- Your extension's content scripts can directly access user data without the need for a background
-  page.
-- A user's extension settings can be persisted even when using [split incognito
-  behavior][incognito].
-- It's asynchronous with bulk read and write operations, and therefore faster than the blocking and
-  serial `localStorage API`.
-- User data can be stored as objects (the `localStorage API` stores data in strings).
-- Enterprise policies configured by the administrator for the extension can be read (using
-  `storage.managed` with a [schema][api-storage]).
+- Content scripts can directly access chrome.storage.
+- Stores JSON serializable values as object properties.
+- Storage is asynchronous with bulk read and write operations.
+- Storage data persists when the user clears the cache and browsing history.
+- Available in the service worker, content scripts, and other extension contexts.
+- A user's extension settings persist even when using [split incognito][incognito].
+- Offers an exclusive read-only [managed storage area][manifest-storage] for enterprise policies.
 
-## Manifest
+{% Details %}
+{% DetailsSummary %}
+💡 Can extensions use Window.localStorage?
+{% endDetailsSummary %}
 
-You must declare the "storage" permission in the [extension manifest][doc-manifest] to use the storage
-API. For example:
+Even though extensions can access [Window.localStorage][mdn-localstorage] in some contexts (popup and other HTML pages), it is not recommended for the following reasons:
+
+- The extension's service worker cannot access the localStorage API.
```

Comment:
```suggestion
- Extension service workers cannot access the web platform's [`localStorage` interface](https://developer.mozilla.org/docs/Web/API/Window/localStorage).
```

Reply:
Not sure I agree with this change since the extension can only have a single service worker. It implies that an extension can have more than one. Hmm... 🤔

Reply:
Glad I found this. I'm currently attempting to update my chrome extension and have found that instructions related to using Storage API could use some updating. I implemented the options example found here https://developer.chrome.com/docs/extensions/reference/storage/#synchronous-response-to-storage-updates but it generated an error

I go to the options page and it gives me a page from 2018 https://developer.chrome.com/docs/extensions/reference/storage/#synchronous-response-to-storage-updates

Also I agree with @AmySteam that sentence does imply that there can be more than one service worker.

Finally have you all consider full examples showing implementations of the various API?

Reply:
@AoverK Thanks for your comments. 

Can you try out the code provided in this PR? Here's a link [synchronous-response-to-storage-updates](https://deploy-preview-3961--developer-chrome-com.netlify.app/docs/extensions/reference/storage#synchronous-response-to-storage-updates). If continue running into problems, can you file an issue with more details?

Great suggestion! Actually, we do have plans to create samples for each API :)